static/js/settings_org.js

```diff
@@ -309,6 +309,7 @@ function insert_tip_box() {
     $(".organization-box").find(".settings-section:not(.can-edit)")
         .not("#emoji-settings")
         .not("#user-groups-admin")
+        .not("#admin-invites-list")
         .prepend(tip_box);
```

Comment:
This should be for a follow-up PR, but we should probably rework this system to be less hacky (e.g. just a template we include in the pages where it's appropriate).

Reply:
sure

Reply:
@timabbott I don't exactly understand your point, I just added this line because it is showing two remainder bar.
![Screenshot_2020-05-01 home - Zulip Dev - Zulip](https://user-images.githubusercontent.com/45326319/80754463-b6456d00-8b4c-11ea-928c-4d2e0cad7b3d.png). to get rid of this I added this line

Reply:
Yeah, what I mean is just that I'm not a fan of this hacky fixup approach to managing bar; I'd prefer something more declarative.